File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/company/dtos/SearchCompaniesResponse.java
package com.slotify.backend.spring.company.dtos;

import com.slotify.backend.spring.company.models.CompanyDocument;
import com.slotify.backend.spring.company.models.CompanyEntity;
import lombok.Builder;
import lombok.Getter;

import java.util.UUID;

@Builder
@Getter
public class SearchCompaniesResponse {
    private UUID companyId;
    private String companyName;
    private String physicalAddress;
    private String description;
    private Short rattingAvg;
    private String s3ImageUrl;
}

File: frontend/app/src/main/java/com/raj/slotify/dtos/company/SearchCompaniesResponse.kt
package com.raj.slotify.dtos.company

import com.google.gson.annotations.SerializedName
import java.util.UUID

data class SearchCompaniesResponse(
    @SerializedName("companyId")
    val companyId: UUID,

    @SerializedName("companyName")
    val companyName: String,

    @SerializedName("physicalAddress")
    val physicalAddress: String,

    @SerializedName("s3ImageKey")
    val s3ImageKey: String?,

    @SerializedName("description")
    val description: String?,

    @SerializedName("rattingAvg")
    val rattingAvg: String?,
)
